public/plugin.ts

```diff
@@ -40,8 +40,13 @@ export class AnomalyDetectionKibanaPlugin
     core.application.register({
       id: 'opendistro-anomaly-detection-kibana',
       title: 'Anomaly Detection',
-      category: DEFAULT_APP_CATEGORIES.kibana,
```

Comment:
thanks for the change, can you also remove the unused import `DEFAULT_APP_CATEGORIES` at line 22 after the only usage is removed?

Reply:
sure. Thanks for reminding.